tests/unit/test_noise_form.py

```diff
@@ -119,7 +126,12 @@ def test_noise_order(mocker, dummy_data, dataset):
         mock_return = (
             dummy_data[["event_type"]]
             if field
-            in [NOISE_TYPES.do_not_respond.name, NOISE_TYPES.omit_row.name, "duplicate_row"]
+            in [
```

Comment:
This change and the one below happen because we generate a configuration for every dataset that has all noise types. This allows us to test that in theory our noise order will always be consistent if we had all noise types in every dataset. But because of this we can not iterate over the row noise types in each dataset.